.github/workflows/format_json.yaml

```diff
@@ -1,5 +1,6 @@
 name: 'PBIX Deserializer'
 description: 'Deserializes PBIX Files into json files for easier version control'
+on: [pull request]
```

Comment:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpull_requestpull_request_targetbranchesbranches-ignore